index.html

```diff
@@ -265,7 +265,13 @@ <h4>Computation steps</h4>
                     <li id="step2B.ii">For each IDREF:
                       <ol>
                         <li id="step2B.ii.a">Set the <code>current node</code> to the node referenced by the IDREF.</li>
-                        <li id="step2B.ii.b">Compute the text alternative of the <code>current node</code> beginning with step 2. Set the <code>result</code> to that text alternative.</li>
+                        <li id="step2B.ii.b">
+                            <ul>
+                                <li>If the <code>current node</code> is <strong>not</strong> <a class="termref">hidden</a>, compute the text alternative of the <code>current node</code> beginning with step 2.</li>
+                                <li>If the <code>current node</code> is <a class="termref">hidden</a>, compute the text alternative of the <code>current node</code> by returning {{HTMLElement/innerText}} of the node.</li>
```

Comment:
```suggestion
                                <li>If the <code>current node</code> is <a class="termref">hidden</a>:
                                  <ol>
                                    <li>If the <code>current node</code> has an <a class="pref">aria-label</a> attribute whose value, when trimmed of white space, is not the empty string, return the value of <a class="pref">aria-label</a>.</li>
                                    <li>Otherwise, compute the text alternative of the <code>current node</code> by returning the {{HTMLElement/innerText}} of the node.</li>
                                  </ol>
                                </li>
```

Reply:
Pending PR change to address @jaragunde's comment in #144